README.md

```diff
@@ -1,27 +1,64 @@
 # GSQ Vocabularies
-## Description
-The Geological Survey of Queensland's (GSQ's) vocabularies, formulated using SKOS, serialised as RDF (turtle) files.
+## Introduction
+The Geological Survey of Queensland (GSQ) publishes vocabularies - a way to describe things and the relationship between things.
```

Comment:
New Line 3: Rewording is much better. But is "things" the best word? Subject? Entity? Concepts? Some combination?

Reply:
From https://www.w3.org/TR/owl-guide 
Every individual in the OWL world is a member of the class owl:Thing.
I think "thing" is ok.